src/Microsoft.Health.Fhir.Core/Features/Search/ISearchService.cs

```diff
@@ -49,14 +49,16 @@ public interface ISearchService
         /// <param name="queryParameters">The search queries.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <param name="isAsyncOperation">Whether the search is part of an async operation.</param>
+        /// <param name="useSmartCompartmentDefinition">Indicates wether to use the expanded SMART on FHIR definition of a compartment</param>
```

Comment:
Missing a period "." at the end of the comment, like in the other comments.

Reply:
Fixed.